src/api/EscargotPublic.h

```diff
@@ -576,6 +576,8 @@ class ESCARGOT_EXPORT ContextRef {
 
     bool initDebugger(const char* options);
     void printDebugger(StringRef* output);
+    StringRef* getClientSource();
```

Comment:
I would use only a single function: `StringRef* getClientSource(StringRef** name);` There is no guarantee that `getClientSourceName` is called.